nifty_pascal-linux/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
```

Comment:
We should try to consolidate down to one file, so we don't have to update multiple gradle build files any time something changes.